crates/hir-def/src/lib.rs

```diff
@@ -128,7 +128,7 @@ impl ModuleId {
     }
 }
 
-/// An ID of a module, **local** to a specific crate
+/// An ID of a module, **local** to a `DefMap`.
 pub type LocalModuleId = Idx<nameres::ModuleData>;
```

Comment:
Am I getting this right?

Reply:
Ye that should be correct now that we have DefMap's for blocks